PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(16:02) electricity rather than the original miners having an advantage and increasing their advantage over time new miners have a technical advantage over existing miners because they buy newer machines with more processing power if you if you bought a mining rig four years ago and i bought one right now i have it i have a bigger advantage or i can capture more coins that's rare that the new entrant has an advantage over the old entrant right so like you you came into it four years ago it's harder for you to capture and
+
+(16:35) mine the coins and it is for me and i think that what that does and when you look at proof of stake it's a little bit different if you had first access to the coins you now have an advantage over somebody who's now just coming in that's trying to capture them and employ them to the network and so from this system bitcoin mining businesses old and new are all in constant competition with each other this is opposite to proof of stake where there is no competition once you have ownership of the tokens you can keep
+
+(17:07) that power in the system forever plus bitcoin developers went to great lengths to make it easy and cheap to run a full node which allows any user to audit the entire blockchain and reject blocks that don't conform with the rules of the network as a result the real power rests with the nodes rather than the miners if miners try to collude and mine invalid blocks the node network simply rejects those blocks so this system is similar to the us constitution where it has three branches of government you would
+
+(17:38) have the executive branch the legislative branch and the judicial branch these branches would have the ability to overrule each other in certain contexts and in addition to this overruling they would have staggered term limits so this is by design you wouldn't want political people to have power forever you would want to vote them out of office in favor of newer people with new ideas and this is how the us works so like the united states the bitcoin network has three branches that limit each other's power the node network has
+
+(18:11) the final say over consensus but relies on miners to order transactions and developers to create new updates that both the miners and the node network accept many other blockchains make multiple trade-offs including making the nodes require immense processing power bandwidth and storage so that only large entities can run them which centralizes the network into a handful of major providers the point i want to make here is that this is also slippery slope right of course like increasing the requirements by 10 maybe ethereum increases by 10 a
```

Comment:
Correggere `10 maybe` in `10 percent maybe` e `10 a year` in `10 percent a year`